Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -112,6 +119,49 @@ YUI.add('ez-translateactionview', function (Y) {
             }
 
             return moreTranslations.join(', ');
+        },
+
+        /**
+         * Tap event handler on New Translation button. It opens language selection box.
+         *
+         * @method _newTranslationLanguageSelectionBox
+         * @private
+         * @param {EventFacade} e
+         */
+        _newTranslationLanguageSelectionBox: function (e) {
+            e.preventDefault();
+            this.fire('languageSelect', {
+                config: {
+                    title: "Select a language for your new translation:",
+                    confirmHandler: Y.bind(this._newTranslation, this),
+                    cancelHandler: null,
+                    canBaseTranslation: true,
+                    existingTranslations: this.get('content').get('currentVersion').getTranslationsList()
+                },
+            });
+        },
+
+        /**
+         * Fires `translate` event after making a selection on LanguageSelectionBox
+         *
+         * @method _newTranslation
+         * @private
+         * @param {EventFacade} e
+         */
+        _newTranslation: function (e) {
+            var languageCode = e.target.get('selectedNewTranslation'),
+                isBaseLanguage = e.target.get('baseTranslation'),
+                data = {
+                    content: this.get('content'),
+                    languageCode: languageCode,
+                    isBaseLanguage: isBaseLanguage
+                };
+
+            if (isBaseLanguage && e.target.get('selectedBaseTranslation') !== null){
+                data.baseLanguageCode = e.target.get('selectedBaseTranslation');
+            }
+
+            this.fire('translate', data);
```

Comment:
you should call this event `translateContent` instead (to be consistent with `editContent`)